ci/test-install/bigtable_install_test.cc

```diff
@@ -57,26 +57,26 @@ int main(int argc, char* argv[]) try {
                                    bigtable::SetCell(family, "c1", "v3")}),
   };
   table.BulkApply(std::move(bulk));
-  std::cout << "bulk mutation successful" << std::endl;
+  std::cout << "bulk mutation successful\n";
 
   auto row0 = table.ReadRow("row-key-0", bigtable::Filter::PassAllFilter());
   if (!row0.first) {
-    std::cout << "Cannot find row-key-0" << std::endl;
+    std::cout << "Cannot find row-key-0\n";
   } else {
     for (auto const& cell : row0.second.cells()) {
       std::cout << cell.row_key() << ": " << cell.family_name() << ":"
                 << cell.column_qualifier() << " = " << cell.value()
-                << std::endl;
+                << "\n";
     }
   }
   auto row1 = table.ReadRow("row-key-1", bigtable::Filter::PassAllFilter());
   if (!row0.first) {
-    std::cout << "Cannot find row-key-0" << std::endl;
+    std::cout << "Cannot find row-key-0\n";
   } else {
     for (auto const& cell : row1.second.cells()) {
       std::cout << cell.row_key() << ": " << cell.family_name() << ":"
                 << cell.column_qualifier() << " = " << cell.value()
-                << std::endl;
+                << "\n";
```

Comment:
Did you rerun clang-format on these files? I'd imagine that a number of these lines may unwrap a little.